File: fizzbuzz.m
function [ res ] = Untitled( fizz, buzz, num )
%UNTITLED Summary of this function goes here
%   Detailed explanation goes here

for i=1:num
	if mod(num, fizz) == 0 && mod(num, buzz) == 0
		res(i,:) = 'FizzBuzz';
	elseif mod(num, fizz) == 0
		res(i,:) = 'Fizz';
	elseif mod(num, buzz) == 0
		res(i,:) =  'Buzz';
	else
		res(i,:) =  num2str(num);
	end
end
end

File: fizzbuzz.lua
function fizzbuzz(fizz, buzz, num)
	if num % fizz == 0 and num % buzz == 0 then
		return "FizzBuzz"
	elseif num % fizz == 0 then
		return "Fizz"
	elseif num % buzz == 0 then
		return "Buzz"
	else return num
	end
end

for i=1,100,1 do
	print(fizzbuzz(3, 5, i))
end
